Announcement-templates.md

```diff
@@ -1,3 +1,31 @@
+## Announcement for ClojureBridge.org
+
+Email the following details to [info@clojurebridge.org](mailto:info@clojurebridge.org) so that we can list the event on the ClojureBridge website:
+
+### Event title
+(For example, "ClojureBridge Edinburgh")
+
+### Location
+Locale (City, Country)
+
+Venue (Full address)
+
+### Description
+A paragraph or two on the event.  Include information on who can register for the workshop (e.g. "Men may attend if accompanied by a woman")
+
+### Additional info
+Further details such as requirement to bring own laptop, schedule and refreshments, and how to register
+
+### Event registration
+URL of registration form or other joining instructions
+
+### Organiser
+GitHub username of main organiser
```

Comment:
Email address of the organizer which needs to be associated with an account with an image at https://gravatar.com/

Github username and twitter handle - both of those are optional.